service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterMapper.java

```diff
@@ -53,10 +61,38 @@ public String mapEncounterToEhrComposition(Encounter encounter) {
             .displayName(buildDisplayName(encounter))
             .originalText(buildOriginalText(encounter));
 
+        final Optional<String> recReference = findParticipantWithCoding(encounter, ParticipantCoding.RECORDER)
+            .map(idMapper::get);
+
+        recReference.ifPresent(encounterStatementTemplateParameters::author);
+
+        messageContext.getInputBundleHolder()
+            .getListReferencedToEncounter(encounter.getIdElement(), CONSULTATION_LIST_CODE)
+            .filter(ListResource::hasDate)
+            .map(ListResource::getDate)
+            .map(Date::toInstant)
```

Comment:
Need to review this conversion of Date to Instant. HAPI does some very strange things with Date and this may represent the system's local time zone

Reply:
yes, I think you'll need to use the `public static String toHl7Format(BaseDateTimeType baseDateTimeType)` version of this method instead.